packages/lib-classifier/src/store/TutorialStore.spec.js

```diff
@@ -334,4 +374,108 @@ describe('Model > TutorialStore', function () {
+
+    it('should reset only the tutorial seen time if mini-course is the kind', function () {
+      const seen = new Date().toISOString()
+      rootStore = RootStore.create({
+        tutorials: TutorialStore.create({ tutorialSeenTime: seen, miniCourseSeenTime: seen }),
+        workflows: WorkflowStore.create()
+      }, {
+        authClient: authClientStubWithoutUser, client: clientStub()
+      })
+
+      rootStore.tutorials.resetSeen('mini-course')
+      expect(rootStore.tutorials.tutorialSeenTime).to.equal(seen)
+      expect(rootStore.tutorials.miniCourseSeenTime).to.be.undefined
+    })
+
+    it('should reset both seen times if there is no defined kind', function () {
+      const seen = new Date().toISOString()
+      rootStore = RootStore.create({
+        tutorials: TutorialStore.create({ tutorialSeenTime: seen, miniCourseSeenTime: seen }),
+        workflows: WorkflowStore.create()
+      }, {
+        authClient: authClientStubWithoutUser, client: clientStub()
+      })
+
+      rootStore.tutorials.resetSeen()
+      expect(rootStore.tutorials.tutorialSeenTime).to.be.undefined
+      expect(rootStore.tutorials.miniCourseSeenTime).to.be.undefined
+    })
+  })
+
+  describe('Actions > setSeenTime', function () {
+    it('should not set the seen time if there is not an active tutorial', function () {
+      rootStore = RootStore.create({
+        tutorials: TutorialStore.create(),
+        workflows: WorkflowStore.create()
+      }, {
+        authClient: authClientStubWithoutUser, client: clientStub()
+      })
+
+      rootStore.tutorials.setSeenTime()
+      expect(rootStore.tutorials.tutorialSeenTime).to.be.undefined
+      expect(rootStore.tutorials.miniCourseSeenTime).to.be.undefined
+    })
+
+    it('should set the seen time for the tutorial kind of tutorial resource', function (done) {
+      rootStore = RootStore.create({
+        tutorials: TutorialStore.create(),
+        workflows: WorkflowStore.create()
+      }, {
+        authClient: authClientStubWithoutUser, client: clientStub()
+      })
+
+      rootStore.workflows.setActive(workflow.id).then(() => {
+        rootStore.tutorials.setActiveTutorial(tutorial.id)
+      }).then(() => {
+        expect(rootStore.tutorials.tutorialSeenTime).to.be.a('string')
+      }).then(done, done)
+    })
+
+    it('should set the seen time for the null kind of tutorial resource', function (done) {
+      rootStore = RootStore.create({
+        tutorials: TutorialStore.create(),
+        workflows: WorkflowStore.create()
+      }, {
+        authClient: authClientStubWithoutUser, client: clientStub(tutorialNullKind)
+      })
+
+      rootStore.workflows.setActive(workflow.id).then(() => {
+        rootStore.tutorials.setActiveTutorial(tutorialNullKind.id)
+      }).then(() => {
+        expect(rootStore.tutorials.tutorialSeenTime).to.be.a('string')
+      }).then(done, done)
+    })
+
+    it('should set the seen time for the null kind of tutorial resource', function (done) {
```

Comment:
'should set the seen time for the minicourse kind of tutorial resource.'

Reply:
Actually there are tutorials with null as the kind, prior to mini-course kinds being added, and we assume those are like standard tutorials, not mini-courses.

Reply:
The expectation here tests `rootStore.tutorials.miniCourseSeenTime` so I think this block is testing minicourses.

Reply:
Ah yes, my mistake. This a typo for this expectation.